app/assets/javascripts/application/multiple-files.js

```diff
@@ -0,0 +1,211 @@
+mumuki.load(function () {
```

Comment:
Cool! Just take in mind that we are now supporting ecma6, so we encourage new code to use `lets`, arrow functions and classes.

Reply:
Migrated to ES6.